motionplan/motionPlanner.go

```diff
@@ -577,24 +567,20 @@ func CheckPlan(
 			if currentTravelDistanceMM > lookAheadDistanceMM {
 				return nil
 			}
-			// If we are working with a PTG plan the returned value for poseInPath will only
-			// tell us how far along the arc we have traveled. Since this is only the relative position,
-			// i.e. relative to where the robot started executing the arc,
-			// we must compose poseInPath with currentPose to get the absolute position.
-			// In both cases we ultimately compose with errorState.
+
+			// Checks for collision along the interpolated route and returns a the first interpolated pose where a collision is detected.
+			interpolatedState := &ik.State{Frame: sf}
 			if relative {
```

Comment:
since we are still working with relative and non-relative plans can we add back this comment
```
// If we are working with a PTG plan the returned value for poseInPath will only
// tell us how far along the arc we have traveled. Since this is only the relative position,
//  i.e. relative to where the robot started executing the arc,
// we must compose poseInPath with segment.StartPosition to get the absolute position.
```